src/session.js

```diff
@@ -5,7 +5,7 @@ let sessions = {};
 function hydrate(request, sessionId) {
   let session = sessions[sessionId];
 
-  Object.assign(request, session, { sessionId });
+  Object.assign(request, { session, sessionId });
```

Comment:
Does this one mean `request.session = session;` and `request.sessionId = sessionId;`?